### DISCUSSION TITLE
Best way to install nightly releases in  an Arch linux based system?

### DISCUSSION THREAD

**Discussion by FurtadoPires** [rustdesk/rustdesk#4407] [General]
I want to install version 1.2.0 due to optimizations in the wayland side, and I would like to know if there's a recommend way to do it without compiling.

I've tried some methods, but had the following problems

-Installing `_pkg.tar.zst_` doesn't make the launchers and can't launch through terminal (probably because of some dependency problem)
-Using `_flatpak_` it doesn't prompt the portal option, so I can't control the host computer
-Using `_AppImage_` it prompts the portal, but still can't control the host
-`Aur` package is using the last release version (1.1.9) which haven't been updated in ages

Therefore, I would like to know a way I could easily install and control a system with gnome Wayland.

**Comment by adamency** [comment]
Hi there, I don't know anything about making it work on Wayland, but I'm here to inform you that there's a nightly Rustdesk package on the AUR. Maybe looking at the PKGBUILD could help you.

**Reply by FurtadoPires** [reply]
The [AUR package](https://aur.archlinux.org/packages/rustdesk) is currently using 1.1.9 version, and also giving errors in the process of install.

I'm currently trying to install the nightly version (1.2.0) which has a lot of fixes for using with Wayland, but for some reason hasn't been promoted to release yet.

**Reply by adamency** [reply]
@FurtadoPires you didn't read my message...

**Comment by nandgator** [comment]
Hi, stable version `1.2.0` has been released. But installing it like so, fails:

```console
$ sudo pacman -U https://github.com/rustdesk/rustdesk/releases/download/1.2.0/rustdesk-1.2.0-0-x86_64.pkg.tar.zst
[sudo] password for us-er: 
:: Retrieving packages...
 rustdesk-1.2.0-0-x86_64                      22.8 MiB  3.41 MiB/s 00:07 [########################################] 100%
error: failed retrieving file 'rustdesk-1.2.0-0-x86_64.pkg.tar.zst.sig' from github.com : The requested URL returned error: 404
warning: failed to retrieve some files
$ sudo pacman -U https://github.com/rustdesk/rustdesk/releases/download/1.2.0/rustdesk-1.2.0-0-x86_64.pkg.tar.zst
[sudo] password for us-er: 
loading packages...
error: '/var/cache/pacman/pkg/rustdesk-1.2.0-0-x86_64.pkg.tar.zst': package missing required signature
```

I raised an issue #4942, but I guess they're tracking discussions now #4331. If @rustdesk, would edit-publish a `.zst.sig` file, it would really help.

> I want to install version 1.2.0 due to optimizations in the wayland side, and I would like to know if there's a recommend way to do it without compiling.

You can use https://github.com/chaotic-aur, they've nightly builds. A binary build may be added as well https://github.com/chaotic-aur/packages/issues/2444.

**Reply by nandgator** [reply]
> You can use https://github.com/chaotic-aur, they've nightly builds. A binary build may be added as well https://github.com/chaotic-aur/packages/issues/2444.

RustDesk is now available on [ChaoticAUR](https://aur.chaotic.cx/), if you've added the repository, you can simply do `sudo pacman -S rustdesk-bin` to install it.

**Comment by FurtadoPires** [comment]
Now that the stable version was updated, the aur version is now being maintained again

**Comment by btakita** [comment]
On chaotic-aur

```
❯ yay -S rustdesk-bin
Sync Explicit (1): rustdesk-bin-1.4.5-1
warning: rustdesk-bin-1.4.5-1 is up to date -- reinstalling
resolving dependencies...
looking for conflicting packages...

Package (1)               Old Version  New Version  Net Change

chaotic-aur/rustdesk-bin  1.4.5-1      1.4.5-1        0.00 MiB

Total Installed Size:  76.10 MiB
Net Upgrade Size:       0.00 MiB

:: Proceed with installation? [Y/n]
(1/1) checking keys in keyring                                                                                                                   [----------------------------------------------------------------------------------------] 100%
(1/1) checking package integrity                                                                                                                 [----------------------------------------------------------------------------------------] 100%
:: File /var/cache/pacman/pkg/rustdesk-bin-1.4.5-1-x86_64.pkg.tar.zst is corrupted (invalid or corrupted package (checksum)).
Do you want to delete it? [Y/n]
error: failed to commit transaction (invalid or corrupted package (checksum))
Errors occurred, no packages were upgraded.
 -> error installing repo packages
~                                                                                                                                                                                                                                   5s 15:49:25
❯ rustdesk --version
Failed to load "librustdesk.so"
libxdo.so.3: cannot open shared object file: No such file or directory
Please run "pacman -Qo libxdo.so.3" to search and install the pkg.```
```
